miio/chuangmi_camera.py

```diff
@@ -12,6 +12,76 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+CONST_HIGH_SENSITIVITY = [
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+    3,
+]
```

Comment:
This is already defined as an enum below, so better simply use something like `[enumvalue] * times` to create the array when needed.

Reply:
Thanks for the advice, fixed it.